### PR TITLE
Tiny clean up packages.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

File: ddl/ddl.go [hunk 1]
```diff
@@ -40,15 +40,15 @@ import (
...
-	// ErrExists returned for creating an exist schema or table.
+	// ErrExists returns for creating an exist schema or table.
 	ErrExists = errors.Errorf("DDL:exists")
-	// ErrNotExists returned for dropping a not exist schema or table.
+	// ErrNotExists returns for dropping a not exist schema or table.
 	ErrNotExists = errors.Errorf("DDL:not exists")
 )
 
-// DDL is responsible for updating schema in data store and maintain in-memory InfoSchema cache.
+// DDL is responsible for updating schema in data store and maintains in-memory InfoSchema cache.
```
Comment (coocood) [inline ddl/ddl.go]: maintaining

File: ddl/ddl.go [hunk 2]
```diff
@@ -66,10 +66,10 @@ type ddl struct {
 	onDDLChange OnDDLChange
 }
 
-// OnDDLChange is used as hook function when schema changed
+// OnDDLChange is used as hook function when schema changed.
 type OnDDLChange func(err error) error
 
-// NewDDL create new DDL
+// NewDDL create a new DDL.
```
Comment (coocood) [inline ddl/ddl.go]: creates

File: ddl/alter.go [hunk 3]
```diff
@@ -28,26 +29,24 @@ const (
...
 	ColumnPositionAfter
 )
 
-// ColumnPosition represent the position of the newly added column
+// ColumnPosition represents the position of the newly added column.
 type ColumnPosition struct {
 	// ColumnPositionNone | ColumnPositionFirst | ColumnPositionAfter
 	Type int
-	// RelativeColumn is the column the newly added column after if type is ColumnPositionAfter
+	// RelativeColumn is the column which is after the newly added column if type is ColumnPositionAfter.
```
Comment (shenli) [inline ddl/alter.go]: It is not right. The column is before the newly added column.

File: kv/union_store.go [hunk 4]
```diff
@@ -22,16 +23,16 @@ import (
...
+type conditionType int
 
 const (
-	// ConditionIfNotExist means the condition is not exist.
-	ConditionIfNotExist SetCondition = iota + 1
-	// ConditionIfEqual means the condition is equals.
-	ConditionIfEqual
-	// ConditionForceSet means the condition is force set.
-	ConditionForceSet
+	// conditionIfNotExist means the condition is not exist.
```
Comment (shenli) [inline kv/union_store.go]: exist is a verb. You should use "is nonexistent".

File: ddl/ddl.go [hunk 5]
```diff
@@ -199,25 +198,25 @@ func setColumnFlagWithConstraint(colMap map[string]*column.Col, v *coldef.TableC
 		for _, key := range v.Keys {
 			c, ok := colMap[strings.ToLower(key.ColumnName)]
 			if !ok {
-				// TODO: table constraint on unknown column
+				// TODO: table constraint on unknown column.
 				continue
 			}
 			c.Flag |= mysql.PriKeyFlag
-			// primary key can not be NULL
+			// primary key can not be NULL.
```
Comment (zimulala) [inline ddl/ddl.go]: primary --> Primary

File: ddl/ddl.go [hunk 6]
```diff
@@ -451,12 +449,12 @@ func (d *ddl) addColumn(ctx context.Context, schema model.CIStr, tbl table.Table
...
+		// Find the mentioned column.
 		c := column.FindCol(cols, spec.Position.RelativeColumn)
 		if c == nil {
 			return errors.Errorf("No such column: %v", name)
 		}
-		// insert position is after the mentioned column
+		// Insert position is after the mentioned column.
 		position = c.Offset + 1
 	}
 	// TODO: Set constraint
```
Comment (zimulala) [inline ddl/ddl.go]: Set --> set ?

File: kv/union_store.go (hunk 4 above)
Comment (siddontang) [inline kv/union_store.go]: "don't exist" may be simpler.

File: ddl/ddl.go (hunk 5 above)
Comment (qiuyesuifeng) [inline ddl/ddl.go]: So many lowercase comments are inner functions, i have to keep them stay the same:(

File: kv/union_store.go (hunk 4 above)
Comment (qiuyesuifeng) [inline kv/union_store.go]: gotcha, `don't exist` is OK.

File: ddl/ddl.go (hunk 6 above)
Comment (qiuyesuifeng) [inline ddl/ddl.go]: gotcha.

File: ddl/ddl.go (hunk 5 above)
Comment (zimulala) [inline ddl/ddl.go]: I see you remove --> Remove, so I think this way too. 
But do it at your will.

Comment (qiuyesuifeng) [inline ddl/ddl.go]: Yes, actually i want, but, maybe in further:(

**Comment by zimulala** [comment]
LGTM

**Pull request merged**